README.md

```diff
@@ -25,6 +26,13 @@ You should use the template to create a new **private** repo in your org, for ex
 
 Once you've created a repository from the template, you'll want to `git clone` your new repo and `cd` into it locally.
 
+Edit the following values in Chart.yaml with your application information:
+- name
+- description
+- appVersion
+- home
+
+note: see Chart.yaml for examples
 
 
```

Comment:
We go from "Edit the chart" to "Install the cli".
What we've really done here is transition from writing a chart to releasing one. They aren't ready to release after editing chart.yaml.

Let's add some instructions here, just think about when you write a new chart roughly what's involved.
* Review your application and gather the information about what you are deploying
* Check the examples for patterns you might want to implement
* Create the values.yaml and any accompanying template files to define 
* Test locally with helm template, repeate

Then talk about releasing and make it clear that's what is happening. Probably a good idea to reference the patterns to point out you can have it done via Github/GitLab if you like.